wagtail/contrib/modeladmin/helpers/url.py

```diff
@@ -28,6 +38,11 @@ def get_action_url_pattern(self, action):
         return self._get_object_specific_action_url_pattern(action)
 
     def get_action_url_name(self, action):
+        if self.base_url_path:
+            return "%s_modeladmin_%s" % (
```

Comment:
This will potentially return an invalid url name. If the base URL path contains slashes, this will be passed into the url name. 

Maybe we should instead use the initialised base URL path as noted above and then convert slashes to underscores for the name here.